scss/components/menu.scss

```diff
@@ -30,24 +31,32 @@ $block: #{$fd-namespace}-menu;
     padding-top: fd-space(base);
     padding-bottom: fd-space(base);
 
-    &__separator>li {
```

Comment:
@ckoutsiaris and @saadmhybris I reopened this because this `&__separator` is an element in BEM and it should be a modifier. An element can standalone whereas a modifier only changes something about the base element. 

I implemented this as a modifier on the list — `fd-menu__list--separated`